### PR TITLE
feat: status tracking

### PR DESCRIPTION
**Pull request by pararm9**
Adding Doc in few minutes

### CODE REVIEW

**Comment by pararm9** [comment]
Merging
as is needed for deployment;
will raise separate PR in seperate PR

File: app/services/whatsapp/broadcast_message_service.rb [hunk 1]
```diff
@@ -21,19 +21,19 @@ def process_audience(audience_label_ids)
       return
     end
     send_message_helper(audience_label_ids)
-    offset += limit
-    Campaign.where(id: campaign.id).update(sent_count: offset)
+
+    Campaign.where(id: campaign.id).update_all('sent_count = sent_count + 50')
```
Comment (taran1898) [inline app/services/whatsapp/broadcast_message_service.rb:25]: Can we use constants here for 50 ? Is this intentional hard coding ?

File: app/services/whatsapp/broadcast_message_service.rb [hunk 2]
```diff
@@ -42,12 +42,12 @@ def send_message_helper(audience_label_ids:)
   def send_message(to:, content:)
     # store in source
     message_id = channel.send_template(to.phone_number, content)
-    return unless message_id.nil?
+    return if message_id.nil?
```
Comment (taran1898) [inline app/services/whatsapp/broadcast_message_service.rb:45]: Is this change intentional? if and unless are opposites.

**Review by taran1898 — APPROVED** [review]
